package.json

```diff
@@ -1,29 +1,37 @@
 {
   "private": true,
   "scripts": {
-    "dev": "prisma generate && next dev",
+    "dev": "prisma generate && node seed.js && next dev",
```

Comment:
Added a seed file that wipes DB and randomly generates 100 new users each time you run project